app/components/card_component.html.erb

```diff
@@ -1,9 +1,12 @@
-<div <%== attributes(classes: card_classes) %>>
-  <%= content %>
+<div class="<%== card_classes_wrapper %>">
+  <% unless content.blank? %>
+    <div <%== attributes(classes: card_classes_content) %>">
```

Comment:
I know there's a reason we use `attributes` instead of `class="#{card_content_classes}"` but for the life of me I cannot remember why...

Also, minor preference for `card_content_classes` vs `card_classes_content`, just because it feels more like `content` is the branch in the "what method am I reaching for?!" decision tree and `classes` is the leaf.